lib/iris/pandas.py

```diff
@@ -116,6 +205,290 @@ def as_cube(pandas_array, copy=True, calendars=None):
     return cube
 
 
+def as_cubes(
+    pandas_array,
```

Comment:
I'm not so sure about the name `pandas_array`. There does appear to be a concept of a `pandas array` which is neither a Dataframe or Series. Instead of `pandas_array` how about: `pandas_data`, `pandas_object`, `pandas_data_structure`, `pandas_dataset` or something else like that?

Reply:
I've gone with `pandas_structure`, based on how the Pandas documentation refers to `Series` and `DataFrame`s collectively ([Intro to data structures](https://pandas.pydata.org/docs/user_guide/dsintro.html)).

5aa5af8883c07720f72cd16fa215bbb044602ab4
e59f86934a5394b2cf9d388c7569d8a95684e038